src/ripple/app/main/Application.cpp

```diff
@@ -1948,6 +1961,22 @@ bool ApplicationImp::updateTables ()
     return true;
 }
 
+void ApplicationImp::setMaxLedger()
+{
+    boost::optional <LedgerIndex> seq;
+    {
+        auto db = getLedgerDB().checkoutDb();
+        *db << "SELECT MAX(LedgerSeq) FROM Ledgers;", soci::into(seq);
+    }
+    if (!seq)
+        maxLedger_= 0;
+    else
+        maxLedger_ = *seq;
```

Comment:
`maxLedger_` is already initialized to 0, so do we need the `else`? `setMaxLedger` is currently only being a called once, but even if we added subsequent calls, it might make sense to leave `maxLedger_` with its previous value.

Otherwise, this could be
```
maxLedger_ = seq ? *seq : 0;
```